suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -77,6 +77,19 @@ tests:
       name: configure client
 
   - test:
+      config:
```

Comment:
Since you are using all default configurations, I think you can rebase code from master to fetch Vasi's changes of adding default values and remove this config from test